deployments/ohw/image/Dockerfile

```
@@ -104,9 +63,6 @@ COPY rsession.conf /etc/rstudio/rsession.conf
 
 USER ${NB_USER}
 
-# FIXME: Split this into two files
-# We can then install things that require python / jupyter separate
-# from scientific python libs - speeding up image build times a lot
 COPY environment.yml /tmp/
 
 RUN conda env update -p ${CONDA_DIR} -f /tmp/environment.yml
```

```suggestion
RUN conda env update --name ${CONDA_ENV} -f /tmp/environment.yml
```

The base PANGEO image's root conda environment is called 'notebook', and available via the 'CONDA_ENV' environment variable. This is what the notebook server is started from, so our notebook related packages (like jupyter-server-proxy or jupyter-rsession-proxy) need to be installed there. I think this is also where the python kernels start by default, so all our scientific python packages also need to be there.